### PR TITLE
tiny fix in key name

### PR DESCRIPTION
**Pull request by antter**

### CODE REVIEW

File: notebooks/preprocess.ipynb [hunk 1]
```diff
@@ -450,7 +450,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.8"
+   "version": "3.8.3"
```
Comment (aakankshaduggal) [inline notebooks/preprocess.ipynb:453]: Is there a reason for this python version change?

**Review by aakankshaduggal — COMMENTED** [review]
/lgtm

File: notebooks/preprocess.ipynb (hunk 1 above)
Comment (antter) [reply notebooks/preprocess.ipynb:453]: No this was not intentional

**Pull request merged**